Apps/Sandcastle/gallery/Custom Projection.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Demonstration of Proj4JS projections.">
+    <meta name="cesium-sandcastle-labels" content="Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        if(typeof require === "function") {
+            require.config({
+                baseUrl : '../../../Source',
+                waitSeconds : 120
+            });
+        }
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+
+var customProjection = new Cesium.CustomProjection('../../SampleData/UserProjection.txt', 'projectionFactory');
```

Comment:
Is there an option to provide the projection code inline here?  That would make this example easier to follow.

Reply:
Not an option right now, but we could possibly allow providing a code string directly, and checking if the input is a url. Wonder what that will do if the user needs external modules though, like for vector math. That needs more evaluation anyway.

Current method definitely at least lets the user build a combined file and then provide a url to that.

Reply:
> Not an option right now, but we could possibly allow providing a code string directly, and checking if the input is a url.

You can just use a text data uri here and the code string will work as is.